src/new/wallet.js

```diff
@@ -399,27 +399,34 @@ class HathorWallet extends EventEmitter {
    * Currently does not have support to send custom tokens, only HTR
    *
    * @param {Array} outputs Array of outputs with each element as an object with {'address', 'value'}
+   * @param {Object} token Token object {'uid', 'name', 'symbol'}. Optional parameter if user already set on the class
    * @param {Array} inputs Array of inputs with each element as an object with {'hash', 'index'}
    *
    * @return {Promise} Promise that resolves when transaction is sent
    **/
-  sendManyOutputsTransaction(outputs, inputs = []) {
+  sendManyOutputsTransaction(outputs, token = null, inputs = []) {
```

Comment:
I wouldn't change the order of the parameters because of compatibility. I think it is better to put as the last parameter.

Reply:
Yes, I totally agree with you. I made this confusion in a merge conflict. Fixed it

Reply:
Thanks! I think it is in use by one of the wallets. Remember to update it there as well, ok?

Reply:
Yes, it was updated!